pkg/kubelet/cm/types.go

```diff
@@ -19,12 +19,15 @@ package cm
 import (
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/utils/cpuset"
 )
 
 // ResourceConfig holds information about all the supported cgroup resource parameters.
 type ResourceConfig struct {
 	// Memory limit (in bytes).
 	Memory *int64
+	// CPU set (number of cpus the cgroup has access to).
```

Comment:
Nit.

```suggestion
	// CPU set (number of CPUs the cgroup has access to).
```

For consistency.

Reply:
fixed!